authentication/authentication-internal-service/src/main/java/io/mosip/authentication/internal/service/integration/RestHelperImpl.java

```diff
@@ -64,25 +65,23 @@
 
 	/** The mosipLogger. */
 	private static Logger mosipLogger = IdaLogger.getLogger(RestHelper.class);
-	
+
 	@Autowired
 	private WebClient webClient;
-	
 
 	@SuppressWarnings("unchecked")
 	@Override
-	public <T> T requestSync(RestRequestDTO request)
-			throws RestServiceException {
+	public <T> T requestSync(RestRequestDTO request) throws RestServiceException {
 		Object response;
 		try {
 			requestTime = DateUtils.getUTCCurrentDateTime();
 			mosipLogger.debug(IdAuthCommonConstants.SESSION_ID, CLASS_REST_HELPER, METHOD_REQUEST_SYNC,
 					"Request received at : " + requestTime);
 			if (request.getTimeout() != null) {
 				response = request(request).timeout(Duration.ofSeconds(request.getTimeout())).block();
-				if(response != null && containsError(response.toString())) {
+				if (response != null && containsError(response.toString())) {
 					mosipLogger.debug(IdAuthCommonConstants.SESSION_ID, CLASS_REST_HELPER, METHOD_REQUEST_SYNC,
-						PREFIX_RESPONSE + response);
+							PREFIX_RESPONSE + response);

```

Comment:
String concatenation is not required now. We can use the placeholders {}. See the example below.
logger.info("test log message with id {} and for user {}", 10, "anusha");

Reply:
@ase-101 , in io.mosip.kernel.core.logger.spi.Logger.debug(String, String, String, String) , the arguments are distinct for different purpose. So we cannot use placeholder as you mentioned. I think this code should be fine.